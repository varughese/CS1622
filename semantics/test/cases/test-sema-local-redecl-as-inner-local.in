void main(void)
{
    int x;
    {
        int x;
    }
}
