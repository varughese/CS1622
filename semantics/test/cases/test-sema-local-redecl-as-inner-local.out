[program
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
      [var-declaration [int] [x]]
      [compound-stmt 
        [var-declaration [int] [x]]
      ]
    ]
  ]
]
